packages/medusa/src/api/routes/admin/discounts/add-region.ts

```diff
@@ -0,0 +1,51 @@
+import { IsNotEmpty, IsString } from "class-validator"
+import { defaultAdminDiscountsFields, defaultAdminDiscountsRelations } from "."
+import { Discount } from "../../../.."
+import DiscountService from "../../../../services/discount"
+import { validator } from "../../../../utils/validator"
+/**
+ * @oas [post] /discounts/{id}/regions/{region_id}
+ * operationId: "PostDiscountsDiscountRegionsRegion"
+ * summary: "Adds Region availability"
+ * description: "Adds a Region to the list of Regions that a Discount can be used in."
+ * x-authenticated: true
+ * parameters:
+ *   - (path) id=* {string} The id of the Discount.
+ *   - (path) region_id=* {string} The id of the Region.
+ * tags:
+ *   - Discount
+ * responses:
+ *   200:
+ *     description: OK
+ *     content:
+ *       application/json:
+ *         schema:
+ *           properties:
+ *             discount:
+ *               $ref: "#/components/schemas/discount"
+ */
+export default async (req, res) => {
+  const { discount_id, region_id } = await validator(
+    AdminPostDiscountsDiscountRegionsRegionParams,
+    req.params
+  )
```

Comment:
As per last Friday, we've decided to not validate params, that are part of the path. Goes for all edited endpoints in the PR.